docs/contributing/index.md

```diff
@@ -1,69 +1,94 @@
 # Contributing
 
-This page explains how to contribute to Autoware.
+First off, thank you for your interest in contributing to Autoware! Like any other open-source project, Autoware can only continue to thrive through the work of people like yourself, and all types of contribution are welcome no matter how big or small.
 
-First, conform to the [Code of conduct](#contribution-workflows) section for the general manners in contributions.
+As a contributor, here are the guidelines that we would like you to follow for Autoware and its associated repositories, all of which are hosted in the [Autoware Foundation Organization](https://github.com/autowarefoundation) on GitHub.
 
-See the [Contribution workflows](#contribution-workflows) section for the workflows.  
-Also, make sure the contribution is in line with the following guidelines:
+- [Code of Conduct](#code-of-conduct)
+- [What Should I Know Before I Get Started?](#what-should-i-know-before-i-get-started)
+  - [Autoware Concepts](#autoware-concepts)
+  - [Contributing to Open Source Projects](#contributing-to-open-source-projects)
+- [How Can I Get Help?](#how-can-I-get-help)
+- [How Can I Contribute?](#how-can-i-contribute)
+  - [Participate in Discussions](#discussions)
+  - [Join a Working Group](#working-groups)
+  - [Report Bugs](#reporting-bugs)
+  - [Make a Pull Request](#pull-requests)
 
-- [Coding guidelines](coding-guidelines/index.md): for contributing source code
-- [Testing guidelines](testing-guidelines/index.md): for contributing tests
-- [Documentation guidelines](documentation-guidelines/index.md): for contributing documents
-- [Pull request guidelines](pull-request-guidelines/index.md): for submitting pull requests
-- [Discussion guidelines](discussion-guidelines/index.md): for discussing with other contributors
-- [Support guidelines](../help/support-guidelines.md): for finding the right channel when you need help
+Like Autoware itself, these guidelines are being actively developed and suggestions for improvement are always welcome! Guideline changes can be proposed by [creating a discussion in the Ideas category](https://github.com/autowarefoundation/autoware/discussions/new?category=ideas).
 
-It is recommended to read the [Contributing pages in ROS 2 Docs](https://docs.ros.org/en/rolling/Contributing.html) as well because Autoware's guidelines generally depend on them.
```

Comment:
I apologise for not explaining the removal - I got mixed up and thought that I was making that change with my updates for the Support pages that is still in progress. 

So, the ROS 2 docs page has a lot of information that is specific to ROS 2, as well as the information about being respectful of prior work etc, but we are asking Autoware contributors to read all of the page anyway. 

If there is something that we want to include in our own contribution guide, then I think that it would be better to explicitly add that content instead of the link.

Reply:
Thank you, agreed!